lib/bot.js

```diff
@@ -170,15 +170,22 @@ class Bot {
   sendToDiscord(author, channel, text) {
     const discordChannelName = this.invertedMapping[channel.toLowerCase()];
     if (discordChannelName) {
-      // #channel -> channel before retrieving:
-      const discordChannel = this.discord.channels.find('name', discordChannelName.slice(1));
+      // #channel -> channel before retrieving and select only text channels:
+      const discordChannel = this.discord.channels.find(
+        (o) => o.name === discordChannelName.slice(1) && o.type === 'text'
```

Comment:
You have to update the stub here: https://github.com/reactiflux/discord-irc/blob/master/test/stubs/discord-stub.js#L13

Since our tests don't actually talk to discord we stub out that part in the tests.
You can stub this out like this:
```js

this.channels = {
  filter() {
    return this.channels;
  },
  get: this.getChannel,
  find: this.getChannel
};